dashboard/app/models/script.rb

```diff
@@ -1665,6 +1665,15 @@ def summarize_for_unit_edit
     summary
   end
 
+  def summarize_for_lesson_edit
+    {
+      isLaunched: script.launched?,
```

Comment:
I tried to rebase off of this PR and i think I am getting an error here because it can't find `script`

Reply:
Good catch. Just updated

Reply:
For some reason its not getScriptData is not working for the new data I added. I'm trying to dig in and figure out why